airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -59,10 +60,18 @@ def get_dag_details(dag_id):
 
 @security.requires_access([(permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG)])
 @format_parameters({'limit': check_limit})
-def get_dags(limit, offset=0):
+def get_dags(limit, sort, offset=0, order_by=None):
```

Comment:
Probably worth doing the following, instead of the `if` statement and since that was the behaviour before this PR, it would be ok, WDYT?
```suggestion
def get_dags(limit, sort="asc", offset=0, order_by=DagModel.dag_id):
```

Reply:
Updated this, however sort is `desc` by default and it's provided by Openapi when not provided by user. So I think leaving it to be provided by openapi is good?

Reply:
Sounds good